name.abuchen.portfolio.tests/src/name/abuchen/portfolio/online/impl/HTMLTableQuoteFeedHistoricalSample.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+        <title>Druckansicht - Titel</title>
+        <meta name="description" content="">
+
+    </head>
+    <body>
+    <div class="container">
+    <header>
+        <h1>Historische Kurse zu DNB Fund - Technology - A EUR ACC (LU0302296495)</h1>
+    </header>
+    <main role="main">
+        <table>
+            <thead>
+            <tr>
+                                    <th class="text-left" width="64px">Datum</th>
+                                    <th >Eröffnung</th>
+                                    <th >Hoch</th>
+                                    <th >Tief</th>
+                                    <th >Schluss</th>
+                                    <th >Volumen</th>
+                            </tr>
+            </thead>
+            <tfoot>
+            <tr>
+                                    <th class="text-left" width="64px">Datum</th>
+                                    <th >Eröffnung</th>
+                                    <th >Hoch</th>
+                                    <th >Tief</th>
+                                    <th >Schluss</th>
+                                    <th >Volumen</th>
+                            </tr>
+            </tfoot>
+            <tbody>
+                            <tr>
+                                    <td class="text-left" width="64px">03.12.2020</td>                                                                                                <td>665,89</td>                                                                            <td>675,00</td>                                                                            <td>665,89</td>                                                                            <td>675,00</td>                                                                                                <td>0</td>                                </tr>
```

Comment:
It really drove me nuts to understand where the quotes are coming from... until I realized I needed to scroll.
I have now formatted the HTML page in the repository. The spaces should not make a difference 😄